client/my-sites/domains/domain-management/list/options-domain-button.jsx

```diff
@@ -83,16 +78,13 @@ class AddDomainButton extends Component {
 
 		return (
 			<Fragment>
-				{ ! config.isEnabled( 'domains/management-list-redesign' ) && (
-					<PopoverMenuItem href={ domainManagementAllRoot() } onClick={ this.trackMenuClick }>
-						{ translate( 'Manage all domains' ) }
-					</PopoverMenuItem>
-				) }
 				<PopoverMenuItem icon="plus" href="/new" onClick={ this.trackMenuClick }>
 					{ translate( 'Add a domain to a new site' ) }
 				</PopoverMenuItem>
 				<PopoverMenuItem icon="create" href="/domains/add" onClick={ this.trackMenuClick }>
-					{ translate( 'Add a domain to a different site' ) }
+					{ selectedSiteSlug
+						? translate( 'Add a domain to a different site' )
+						: translate( 'Add a domain to an existing site' ) }
```

Comment:
It makes more sense to use `...to an existing site` from the All Domains list since there isn't a specific site already selected.